openupgrade_scripts/scripts/stock_dropshipping/16.0.1.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,6 @@
+---Models in module 'stock_dropshipping'---
+---Fields in module 'stock_dropshipping'---
+---XML records in module 'stock_dropshipping'---
+DEL stock.location.route: stock_dropshipping.route_drop_shipping (noupdate)
+NEW stock.route: stock_dropshipping.route_drop_shipping (noupdate)
+# Nothing to do.
```

Comment:
For other times, please indicate always why nothing to do. In this case, because the model has been renamed in `stock`.